### DISCUSSION TITLE
gumroad issue

### DISCUSSION THREAD

**Discussion by kitzler-walli** [kimai/kimai#2645] [Questions & Answers] [answered]
I just tried to buy a plugin via gumroad but I didn't receive a mail (not even a spam mail) and I was not able to generate an invoice. Clicking the generate button just redirected to the plugin overview.

How to get any help on this?

**Comment by kevinpapst** [comment, marked as answer]
Normally via gumroad support email, but you can contact me as well at info@keleo.de
